### DISCUSSION TITLE
NebulaLogger can't co-exist with Neuraflash Einstein Insights?

### DISCUSSION THREAD

**Discussion by ibmbw** [jongpie/NebulaLogger#709] [Q&A] [answered]
Nebula Logger isn't able to be installed when client org has another package for Einstein Insights. 
Is there any way around this? TIA
![Screenshot 2024-07-05 at 5 24 10 PM](https://github.com/jongpie/NebulaLogger/assets/174779019/82e58093-f725-4e9f-8077-5467c5831a5e)

**Comment by jamessimone** [comment, marked as answer]
@ibmbw all packages have advanced options for installation - either through the CLI or the package install web page - that allow you to dictate the Apex Compile option to be strictly for the classes in the package itself, instead of all classes in the org. Compiling only the apex classes in Nebula Logger should allow you to proceed.
